src/client/src/components/form/formUtils.ts

```diff
@@ -58,6 +49,50 @@ export const getFieldBorderColor = (isReadOnly: boolean) => {
   };
 };
 
+/**
+ * Formats a number with scientific notation if the number is less than 0.001 and has more than 3 decimal places.
```

Comment:
REQ: _if the number is less than 0.001 and has more than 3 decimal places_ ist hier überflüssig.
```suggestion
 * Formats a number with scientific notation.
```